### PR TITLE
Separate run script to build and run part

### PR DESCRIPTION
**Pull request by chaspy**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -eu
+docker-compose build
+docker-compose run web bin/rake db:create db:migrate db:seed
```
Comment (kamontia) [inline build.sh:4]: Add `yarn install` command to install depending libraries

Comment (kamontia) [reply build.sh:4]:
```suggestion
docker-compose run web bin/rake db:create db:migrate db:seed
yarn install
```

**Comment by chaspy** [comment]
@kamontia 
Please review and merge

**Comment by kamontia** [comment]
@chaspy Please rebase master to avoid `undefined method id: for nil:NilClass`

**Pull request merged**